misk/src/test/kotlin/misk/web/metadata/protos/Robot.kt

```diff
@@ -0,0 +1,147 @@
+// Code generated by Wire protocol buffer compiler, do not edit.
+package misk.web.metadata.protos
+
+import com.squareup.wire.FieldEncoding
+import com.squareup.wire.Message
+import com.squareup.wire.ProtoAdapter
+import com.squareup.wire.ProtoReader
+import com.squareup.wire.ProtoWriter
+import com.squareup.wire.Syntax.PROTO_2
+import com.squareup.wire.WireField
+import com.squareup.wire.`internal`.sanitize
+import kotlin.Any
+import kotlin.Boolean
+import kotlin.Int
+import kotlin.Long
+import kotlin.String
+import kotlin.hashCode
+import kotlin.jvm.JvmField
+import okio.ByteString
+
+public class Robot(
```

Comment:
Do these classes need to be checked in or could they live in protos folder and be generated when tests are run? Seems like they could grow stale and lose effectiveness if Wire changes the generated code etc.

Reply:
Yeah definitely agree :) I was hoping to have them compiled but when I paired with @swankjesse he said this was probably best